arangod/RestHandler/RestDocumentHandler.cpp

```diff
@@ -193,7 +193,7 @@ RestStatus RestDocumentHandler::insertDocument() {
                          opOptions.isSynchronousReplicationFrom);
 
   // find and load collection given by name or identifier
-  _activeTrx = createTransaction(cname, AccessMode::Type::WRITE);
+  _activeTrx = createTransaction(cname, AccessMode::Type::WRITE, opOptions);
```

Comment:
isn't the `addHint(IS_FOLLOWER_TRX)` a couple of lines be below now twice? 
I would prefer to have it centralized

Reply:
Changed.